docs/contracts/functions.rst

```diff
@@ -70,18 +70,6 @@ with two integers, you would use something like the following::
 
 Function parameters can be used as any other local variable and they can also be assigned to.
 
-.. note::
-
-  An :ref:`external function<external-function-calls>` cannot accept a
-  multi-dimensional array as an input
-  parameter. This functionality is possible if you enable the ABI coder v2
-  by adding ``pragma abicoder v2;`` to your source file.
-
-  An :ref:`internal function<external-function-calls>` can accept a
-  multi-dimensional array without enabling the feature.
```

Comment:
I'm not sure we want to remove it yet. ABI coder v2 is now the default but v1 is still available. Maybe it's better to just rewrite it?

```markdown
  An :ref:`external function<external-function-calls>` cannot accept a
  multi-dimensional array as an input parameter if you are using ABI coder v1.
```

Reply:
Sounds good!

Reply:
> @smarteasy Thanks for helping us keep the docs up to date!
> 
> The parts you removed are indeed a bit out of date but the old ABI coder has not been removed yet and I think there's some useful info that can be salvaged from them. I added my suggestions in review comments.

What do I have to do to live up to your opinion?

Reply:
You could put back the note and use the text I proposed. Or modify it if you'd prefer to word it differently.

Basically, you're right that the current text is incorrect (or at least misleading) on 0.8.0 - the ABI coder v2 is enabled by default after all - but the information about the differences between v1 and v2 is still useful and we'd like to keep it in some form.